src/modules/nfts/components/wizard/RoyaltiesCreators.tsx

```diff
@@ -0,0 +1,706 @@
+          ref={ref}
+          onChange={(value) => updateCreator(creatorAddress, value as number)}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter') {
+              setShowPercentageInput(false);
+            }
+          }}
+          // controls={false} // not supported in this version of antd, upgrade?
+        />
+      ) : (
+        <Paragraph
+          onClick={() => !isHolaplex && setShowPercentageInput(true)}
+          style={{ marginRight: 5, fontSize: 14, cursor: isHolaplex ? '' : 'pointer' }}
+        >
+          {share.toFixed(2).replace(/[.,]00$/, '')}%
+        </Paragraph>
+      )}
+      {isHolaplex ? (
+        <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <g opacity="0.5">
+            <path
+              d="M6.33333 3.66663H1.66667C1.29848 3.66663 1 3.9651 1 4.33329V6.66663C1 7.03482 1.29848 7.33329 1.66667 7.33329H6.33333C6.70152 7.33329 7 7.03482 7 6.66663V4.33329C7 3.9651 6.70152 3.66663 6.33333 3.66663Z"
+              fill="white"
+            />
+            <path
+              d="M2.33333 3.66663V2.33329C2.33333 1.89127 2.50893 1.46734 2.82149 1.15478C3.13405 0.842221 3.55797 0.666626 4 0.666626C4.44203 0.666626 4.86595 0.842221 5.17851 1.15478C5.49107 1.46734 5.66667 1.89127 5.66667 2.33329V3.66663"
+              stroke="white"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            />
+          </g>
+        </svg>
+      ) : (
+        <RemoveCreatorIcon onClick={() => removeCreator(creatorAddress)} />
+      )}
+      {isHolaplex && (
+        <Modal
+          width={668}
+          visible={isModalVisible}
+          onOk={() => setIsModalVisible(false)}
+          onCancel={() => setIsModalVisible(false)}
+          closeIcon={
+            <StyledCloseIcon viewBox="0 0 24 24">
+              <path
+                d="M18 6L6 18"
+                stroke="white"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+              <path
+                d="M6 6L18 18"
+                stroke="white"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+            </StyledCloseIcon>
+          }
+          footer={null}
+          bodyStyle={{
+            borderRadius: '10px',
+            background: '#1a1a1a',
+            padding: '114px 67px 47px 67px',
+          }}
+        >
+          <FeesModalContent />
+        </Modal>
+      )}
+    </StyledCreatorsRow>
+  );
+};
+
+export default function RoyaltiesCreators({
+  previousStep,
+  goToStep,
+  images,
+  dispatch,
+  nextStep,
+  form,
+  userKey,
+  formValues,
+  setDoEachRoyaltyInd,
+  doEachRoyaltyInd,
+  index,
+  isFirst = false,
+}: Props) {
+  const [creators, setCreators] = useState<Array<Creator>>([
+    { address: process.env.NEXT_PUBLIC_HOLAPLEX_PUBKEY ?? '', share: 2 },
```

Comment:
good to bump up into a constant.

Reply:
As long as the object is frozen - don't want code accidentally mutating the values if it's a global object.

Reply:
How about something like this
```
const HOLAPLEX_CREATOR = Object.freeze({
  address: process.env.NEXT_PUBLIC_HOLAPLEX_HOLDER_PUBKEY ?? '',
  share: 2,
});
```